### DISCUSSION TITLE
i20230310 - No sound after latest pacman -Syu (2023-03-29)

### DISCUSSION THREAD

**Discussion by kw-sudo** [rern/rAudio#1284] [Fixed Issues]
Yesterday I've flashed a new microSD card and did a clean install from the scratch. I could verify the following:

- image 20230212 + addon update 20230310 --> it works
- plus pacman -Syu --> After adding new music the progress bar (play) shows activity but no sound

`System settings: 'straight' USB2Amp connection ('bitperfect settings'), all onboard and GPIO devices are disabled.`

`Player settings: no volume control, everything else is disabeld
`
```
MPD settings:

mpd 0.23.12-1
# /srv/http/data/mpdconf/mpd.conf
bind_to_address     "0.0.0.0"
db_file             "/srv/http/data/mpd/mpd.db"
music_directory     "/mnt/MPD"
playlist_directory  "/srv/http/data/playlists"
state_file          "/var/lib/mpd/mpdstate"
user                "mpd"
input {
	plugin  "curl"
}
audio_output {
	name           "nubert xCore USB Audio"
	device         "hw:0,0"
	type           "alsa"
	auto_resample  "no"
	auto_format    "no"
	mixer_type     "none"
}

mpd.service- Music Player Daemon
     Loaded: loaded (/usr/lib/systemd/system/mpd.service; disabled; preset: disabled)
    Drop-In: /usr/lib/systemd/system/mpd.service.d
             └─00-arch.conf
             /etc/systemd/system/mpd.service.d
             └─override.conf
     Active: active (running) since Thu 2023-03-30 10:10:14 CEST; 24min ago
       Docs: man:mpd(1)
             man:mpd.conf(5)
   Main PID: 520 (mpd)
      Tasks: 3 (limit: 1060)
        CPU: 1.600s
     CGroup: /system.slice/mpd.service
             └─520 /usr/bin/mpd --systemd /srv/http/data/mpdconf/mpd.conf
```

```
rAudio 20230310
6.1.11-1 aarch64
Raspberry Pi 3B Rev 1.2
BCM2837 • 951 MiB
4 x Cortex-A53 @ 1.20 GHz

firmware-raspberrypi 20230125-1
linux-firmware 20230210.bf4115c-1
linux-firmware-whence 20230210.bf4115c-1
raspberrypi-bootloader 20230324-1
raspberrypi-firmware 20230322-1
```

**Comment by klassikmann** [comment]
@master-kw 
I have no direct help, but I saw you had a similar problem last October (https://github.com/rern/rAudio-1/discussions/1066).
What is `amixer info` and `aplay -l` reporting?

**Reply by kw-sudo** [reply]
PS: actually I'm on i20230212+addon 20230310 so I can listen in any way

amixer info
```
Card default 'Audio'/'nubert nubert xCore USB Audio at usb-3f980000.usb-1.2, high speed'
  Mixer name	: 'USB Mixer'
  Components	: 'USB20b1:3091'
  Controls      : 6
  Simple ctrls  : 2
```

aplay -l
```
**** List hardware devices (play) ****
Karte 0: Audio [nubert xCore USB Audio], Gerät 0: USB Audio [USB Audio]
  Sub-Geräte: 1/1
  Sub-Gerät #0: subdevice #0
```

PPS: I make pacman -Syu updates regularly, last Monday it still worked fine, with the Wednesday's upate the issue arrived

**Reply by kw-sudo** [reply]
Another try with the today's update, no changes. Going back to 'working standard'.

**Comment by kw-sudo** [comment]
With the today's update (_raspberrypi-bootloader 20230403-1_) it works again.

**Comment by klassikmann** [comment]
Mysterious

**Reply by kw-sudo** [reply]
`'Insanity is doing the same thing over and over again, but expecting different results'." `

That's exactly how I felt the last few days. ;-) 

The new bootloader seems to be the only change (OS/boot files). The only solution for listening - until now - was a standard r/Audio installation.
